fastcov.py

```diff
@@ -486,7 +486,7 @@ def exclProcessSource(fastcov_sources, source, exclude_branches_sw, include_bran
                     del fastcov_data["branches"][i]
 
             # Skip to next line as soon as possible
-            if "LCOV_EXCL" not in line:
+            if (not exclude_line_marker and "LCOV_EXCL" not in line) or not any(val in line for val in exclude_line_marker):
```

Comment:
```
if not any(marker in line for marker in exclude_line_marker):
    continue
```

Reply:
same as above

Reply:
True. This conditional is a bit unwieldy as is. I vote we break it up similar to how I proposed above:

```python
            # Skip to next line as soon as possible
            markerFound = False
            for marker in exclude_line_marker + ["LCOV_EXCL"]:
                if marker in fileStr:
                    markerFound = True
                    break

            if not markerFound:
                continue
```

In fact, we could just make a function:
```python
            def containsMarker(markers, strBody):
                for marker in markers:
                    if marker in strBody:
                        return True
                return False
```

we we could call like:
```python
if not containsMarker(exclude_line_marker + ["LCOV_EXCL"], line)
    continue
```